conf/application.conf

```diff
@@ -119,7 +117,15 @@ microservice {
       host = localhost
       port = 8500
     }
+
+    object-store {
+      host = "localhost"
+      port = 8464
+    }
   }
 }
 
 jdk.xml.maxOccurLimit = 100000
+
+object-store.default-retention-period = 1-month
```

Comment:
Shouldn't the retention period be 7 years?

Reply:
Actually, don't think this is needed at all.